codacy/values.yaml

```diff
@@ -1,10 +1,11 @@
 global:
   minio:
     create: true
-    location: codacy-minio.codacy.svc.cluster.local
+    location: codacy-minio
     port: 9000
     accessKey: "AKIAIOSFODNN7EXAMPLE"
     secretKey: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+    fullnameOverride: codacy-minio
```

Comment:
I don't think this will work. Minio's chart doesn't read values from here 

https://github.com/helm/charts/blob/master/stable/minio/templates/_helpers.tpl#L15

Reply:
This should be in `minio.fullnameOverride` down there